connector.start.exception = Connector [{0}] could not be started successfully. Reason:[{1}]
connector.delete.exception = Connector [{0}] could not be deleted. Reason:[{1}]
